pom.xml

```diff
@@ -2012,12 +2012,12 @@
     <profiles>
         <profile>
             <id>grm</id>
-            <!-- comment out activation section to turn this profile off permanently-->
+            <!-- comment out activation section to turn this profile off permanently
```

Comment:
either that or tell them to build with mvn clean install -Dgrm.off each and every time